contracts/DevDependencies.sol

```diff
@@ -8,6 +8,4 @@ pragma solidity ^0.5.0;
 import "openzeppelin-solidity/contracts/token/ERC20/ERC20Mintable.sol";
 import "solidity-multicall/contracts/MultiCaller.sol";
 
-
-contract DevDependencies {  // solhint-disable no-empty-blocks
```

Comment:
How are you avoiding this error?

Reply:
Setting "no-empty-blocks": "off" in the solhint configure file

Reply:
OK... Are we sure we want to universally avoid this warning?